packages/common/src/api/tan-query/types.ts

```diff
@@ -1,7 +1,9 @@
+import { DefinedInitialDataOptions } from '@tanstack/react-query'
+
 /**
  * Standard tan-query pass-thru options that we use
  */
-export type QueryOptions = {
-  staleTime?: number
-  enabled?: boolean
-}
+export type Config = Pick<
+  DefinedInitialDataOptions<any>,
```

Comment:
I know it might be a little daunting, but we should explore actually using the types for this correctly. I think all of the hooks know what kind of data they are returning, so this isn't _too_ bad to do.